tests/testthat/test_mz_summary.R

```diff
@@ -0,0 +1,72 @@
+## create toy example data set
+data("mat_test", package = "MetNet")
+colnames(mat_test) <- paste0("intensity_", 1:20)
+mz <- c(100, 150, 262.0528, 262.0528, 262.0528, 348.0532, 448.0532)
+rt <- c(100, 100, 50, 150, 150, 150, 150)
+mat_test <- cbind(mz = mz, rt = rt, mat_test)
+
+## transformations object for structural calculation
+transformations <- rbind(
+  c("Malonyl group (-H2O)", "C3H2O3", 86.0003939305, "+"),
+  c("Monosaccharide (-H2O)", "C6H10O5", 162.0528234315, "-"))
+transformations_neg <- transformations <- data.frame(
+  group = transformations[, 1],
+  formula = transformations[, 2],
+  mass = as.numeric(transformations[, 3]),
+  rt = transformations[, 4])
+transformations_neg[, 3] <- -1 * transformations_neg[, 3]
+
+
+## structural calculation
+struct_adj <- structural(mat_test,
+                         transformation = transformations, ppm = 5, directed = FALSE)
+
+struct_adj_neg <- structural(mat_test,
+                             transformation = transformations_neg, ppm = 5, directed = FALSE)
+
```

Comment:
could you also add a test where `directed = FALSE` and check if `mz_summary` returns "only half the numbers"?